web-app/src/components/jobs/JobPostPreview.tsx

```diff
@@ -0,0 +1,180 @@
+
+import React from "react";
+import styled from "styled-components";
+import { JobPostInterface } from "../../pages/show-jobs/show-jobs";
+import RatingsDropdown from "../ratings-dropdown";
+import { Button } from "src/components/button";
+import dollarImage from "../../images/job-post-icons/dollar.png";
+import addressImage from "../../images/job-post-icons/address.png";
+import jobTypeImage from "../../images/job-post-icons/job-type.png";
+import shiftsImage from "../../images/job-post-icons/shifts.png";
```

Comment:
Import all of these images in one line instead of having separate import statements for each image

Reply:
in one line you mean like this
import  {dollarImage ,addressImage ,jobTypeImage ,shiftsImage } from "./images/"